1.8/usage/jobs/getting-started.md

```diff
@@ -27,23 +29,127 @@ From the DC/OS web interface, click the **Jobs** tab, then the **New Job** butto
 
 From the **Jobs** tab, click the name of your job to modify or delete your job. While the job is running you can click the job instance to drill down to **Details**, **Files**, and **Logs** data.
 
+# DC/OS CLI
+  
+You can create and manage jobs from the DC/OS CLI using `dcos job` commands. To see a full list of available commands, run `dcos job --help`.
+ 
+## Add a Job
+ 
+1. Create a job file in JSON format:
+ 
+    ```json
+    {
+        "id": "myjob",
+        "description": "A job that sleeps regularly",
+        "run": {
+            "cmd": "sleep 20000",
+            "cpus": 0.01,
+            "mem": 32,
+            "disk": 0
+        },
+        "schedules": [
+            {
+                "id": "sleep-schedule",
+                "enabled": true,
+                "cron": "20 0 * * *",
+                "concurrencyPolicy": "ALLOW"
+            }
+        ]
+    }
+    ```
+ 
+     **Note:** You can only assign one schedule to a job.
+ 
+1. Add the job:
+    ```bash
+    dcos job add <myjob>.json
+    ```
+ 
+    **Note:** You can choose any name for your job file.
+ 
+1. Go to the "Jobs" tab of the DC/OS web interface to verify that you have added your job, or verify from the CLI:
+    ```bash
+    dcos job list
+    ```
+ 
+## Schedule-Only JSON
+ 
+If you use the same schedule for more than one job, you can create a separate JSON file for the schedule. Use the [`dcos job schedule add` command](#add-sched) associate a job with the schedule.
+ 
+```json
+{
+    "concurrencyPolicy": "ALLOW",
+    "cron": "20 0 * * *",
+    "enabled": true,
+    "id": "nightly",
+    "nextRunAt": "2016-07-26T00:20:00.000+0000",
+    "startingDeadlineSeconds": 900,
+    "timezone": "UTC"
+}
+```
+ 
+## Remove a Job
+ 
+1. Enter the following command on the DC/OS CLI:
+ 
+    ```
+    dcos job remove <job-id>
+    ```
+ 
+1. Go to the "Jobs" tab of the DC/OS web interface to verify that you have removed your job, or verify from the CLI:
+ 
+    ```
+    dcos job list
+    ```
+ 
+## Modify or View a Job
+ 
+To modify your job, by update your JSON job file, then run
+ 
+```
+dcos job update <job-file.json>
+```
+
+<a name="add-sched"></a>
+To modify the schedule of your job, modify the `schedules` parameter of your jobs file. Then, run one of the following commands:
+ 
+```bash
+dcos job schedule add <job-id> <schedule-file>.json
+dcos job schedule remove <job-id> <schedule-id>
+dcos job schedule update <job-id> <schedule-file>.json
```

Comment:
This command doesn't work for me. I get:
```
calico:~ suzanne$ dcos job schedule update myjob second-sched.json 
Traceback (most recent call last):
File "cli/dcoscli/subcommand.py", line 99, in run_and_capture
File "cli/dcoscli/job/main.py", line 28, in main
File "cli/dcoscli/util.py", line 21, in wrapper
File "cli/dcoscli/job/main.py", line 41, in _main
File "dcos/cmds.py", line 43, in execute
File "cli/dcoscli/job/main.py", line 514, in _update_schedules
KeyError: 0
```

I also wasn't able to update via the API (I didn't see a change):
```
calico:~ suzanne$ curl -X PUT -H "Content-Type: application/json" -H "Authorization: token=$(dcos config show core.dcos_acs_token)" $(dcos config show core.dcos_url)/service/metronome/v1/schedules -d@/Users/suzanne/schedule.json
```